private-templates-service/client/src/store/templates/reducer.ts

```diff
@@ -0,0 +1,32 @@
+import { TemplateState, TemplateAction, REQUEST_TEMPLATES_GET, REQUEST_TEMPLATE_GET_SUCCES, REQUEST_TEMPLATE_GET_FAIL } from "./types";
+import { TemplateList } from "adaptive-templating-service-typescript-node";
+import { ActionsConfig } from "adaptivecards";
+
+const initialState: TemplateState = {
+  fetching: false,
+  templates: new TemplateList(),
+}
+
+export function templateReducer(state = initialState, action: TemplateAction): TemplateState {
```

Comment:
Can we name this something different so that we know its a reducer for multiple templates? i'm creating one (currentTemplateReducer) where we are managing the state for one template, the template on the preview page or designer so i'm just thinking this might get confusing.

this may be an opportunity to name this `src/store/template` directory and the files inside in a more descriptive way

Reply:
Sorry not files but the exported members of the files

Reply:
Renamed!